Modules/Scripted/Endoscopy/Endoscopy.py

```diff
@@ -144,6 +145,7 @@ def setupFlythroughUI(self):
         flythroughCollapsibleButton = ctk.ctkCollapsibleButton()
         flythroughCollapsibleButton.text = _("Flythrough")
         flythroughCollapsibleButton.enabled = False
+        flythroughCollapsibleButton.collapsed = False
```

Comment:
We minimize the number of click required and not collapse this.

Reply:
Ditto. In that case, this is not needed as the default for collapsible button is to be "open"